docs/releasenotes/index.rst

```diff
@@ -9,6 +9,12 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+v1.0.2
+======
+
+* The "on-failure" and "always" restart policies for containers have been disabled.
```

Comment:
ANd maybe link to issue here too, so people who are interested can track progress instead of maybe filing new issues saying "I need this".